src/main/features/response/routes/evidence.ts

```diff
@@ -20,8 +19,7 @@ function renderView (form: Form<Evidence>, res: express.Response): void {
 
   res.render(page.associatedView, {
     form: form,
-    claimantName: claim.claimData.claimant.name,
-    allEvidenceTypes: EvidenceType.all()
+    claimantName: claim.claimData.claimant.name
```

Comment:
claim.claimData.claimant.name can be accessed directly on the view { claimantName: claim.claimData.claimant.name }

Reply:
It's going to be removed in next iteration as information about claimant name is not displayed anymore in the current version of prototype.

I'd leave it for 2-3 days as it is.